statsd/client.go

```diff
@@ -269,14 +269,13 @@ func (s *Client) SetPrefix(prefix string) {
 	s.prefix = prefix
 }
 
-// NewSubStatter returns a SubStatter with appended prefix
-func (s *Client) NewSubStatter(prefix string) SubStatter {
+// NewSubStatter returns a SubStatter with an updated prefix that equals the
+// old prefix with 'newScope' appended.
+func (s *Client) NewSubStatter(newScope string) SubStatter {
```

Comment:
I would prefer to keep the old method signature here (old doc string and arg name).